app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -65,6 +64,7 @@ You do not need to wait until you have a confirmed offer for a teacher training
 
 If you choose a teacher training course that has a salary you're not eligible for any student finance.
 
+$teacherpay$
```

Comment:
I'm not sure this makes sense here - we haven't mentioned pay at all so you can't find out **more** about it :-D

Plus I don't think it really fits?

If the related content link has a big CTR, is there a better way to fit it into the page? Or do we just accept that it doesn't fit?